pinot-core/src/main/java/com/linkedin/pinot/core/operator/MCombineGroupByOperator.java

```diff
@@ -49,12 +49,14 @@
 public class MCombineGroupByOperator extends BaseOperator {
```

Comment:
We have 2 combine operators, one for aggregation only and selection, the other for aggregation group-by. I have a plan to merge them into a single operator (there is a TODO on that), and will address in another PR.